src/ecrc-frontend/src/components/page/informationreview/InformationReview.js

```diff
@@ -462,12 +484,11 @@ export default function InformationReview({
             />
             <span className="declaration-cb">
               I certify that, to the best of my knowledge, the information I
-              have provided on my application and will provide as necessary is
-              complete, honest and accurate. I understand that a false statement
-              or omission of facts herein may lead to a denial of a cannabis
-              workers registration. I am also aware that later discovery of an
-              omission or misrepresentation may be grounds for any finding of
-              suitability to be suspended or revoked.
+              have provided on my application is complete, honest and accurate.
+              Any false statements or deliberate omissions on a consent form
+              filed with the CRRP may result in the inability of the CRRP to
+              accurately determine whether the applicant poses a risk to
+              children or vulnerable adults.
```

Comment:
hmm I am surprised they say this... seems like it is revealing good news to malicious people with bad intentions 😄

Reply:
Does feel like it needs a "and will result in application denial" or something at the end.

Reply:
yeah I would maybe just check with them and see what they think about adding that... might be a good idea. not a blocker obviously for this task anyways